src/seo.ts

```diff
@@ -96,6 +95,19 @@ export class Seo extends BasePlugin<SEOConfig> {
     window.parent.postMessage({ type: 'SEOStructuredData', SEOStructuredData }, '*');
   }
 
+  private static concatenateStartTimeQueryParam(url: string, newParamName: string, newParamValue: number): string {
+    const encodedParamValue = encodeURIComponent(newParamValue);
+    const separator = url.includes('?') ? '&' : '?';
+    const fragmentIndex = url.indexOf('#');
```

Comment:
does it possible if url contains few `#`?

Reply:
no